### ISSUE TITLE
installing pydicom

### ISSUE THREAD

**Issue by yidawang** [scitran/reaper#7]
Hi,

In your instructions there's one line for installing pydicom as "pip install git+https://github.com/scitran/pydicom.gitmirror.git@value_mismatch". I encounter an error when running this line. Even if I remove "mirror.git" or "mirror.git@value_mismatch", I couldn't get through it. Any ideas about how to fix this?

Thanks!

**Comment by gsfr** [comment]
Sorry, the instructions are out of date. Please try this:

```
pip install git+https://github.com/scitran/pydicom.git@0.9.9_value_vr_mismatch
```

Make sure to start with a clean virtualenv. The error you reported in #8 is probably due to having installed a more recent and incompatible version of pydicom, which now in fact gets imported as "pydicom".

**Comment by yidawang** [comment]
Thanks! It helps me get through the dicom module error, but I got another related one at next line ("import scitran.data.medimg.pfile as scipfile").
File "xxx/data/scitran/data/medimg/dcm/dcm.py", line 185, in <module>
    ignore_rules=[dcmstack.extract.ignore_non_ascii_bytes],
AttributeError: 'module' object has no attribute 'ignore_non_ascii_bytes'

Looks like it's still a dicom related issue? Thanks again.

**Comment by gsfr** [comment]
It looks like that function was removed a few weeks ago:
https://github.com/moloney/dcmstack/commit/3911feb357c47854b817ddc85a7052caddc71e49#diff-111bbd48f0ac197d9050fd057de6577eL26
For now, you might want to use the version just before that commit, i.e., this one:
https://github.com/moloney/dcmstack/commit/6d49fe01235c08ae63c76fa2f3943b49c9b9832d

**Comment by yidawang** [comment]
Solved. Thanks so much!

**Comment by gsfr** [comment]
This also closes #8.
